### PR TITLE
feat/new error types

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/common/web/statics/locale/messages_en.json [hunk 1]
```diff
@@ -465,6 +465,31 @@
...
+	"UploadAudio_Error_Unsupported_Audio_Codec": "The {{codec}} audio codec used to make this file isn't supported and it can't be uploaded. Try a different file.",
+	"UploadVideo_Error_MP4_Unsupported_Format": "This file is not in a supported MP4 format and can’t be uploaded. Try a different file.",
+	"UploadVideo_Error_Unsupported_Sample_Rate": "This video file is not in a supported sample rate and can’t be uploaded. Try a different file.",
+	"UploadVideo_Error_Unsupported_Bit_Rate": "The video bit rate {{fps}} is too high and file can't be uploaded. Try a different file.",
+	"UploadVideo_Error_Video_Audio_Duration_Mismatch": "The file's video length {{videoDuration}} & audio length {{audioDuration}} don't match. Upload the file again.",
+	"UploadVideo_Error_Unsupported_Video_Codec": "The {{codec}} video codec used to make this file isn't supported and it can't be uploaded. Try a different file.",
+	"UploadVideo_Error_Video_Duration_SiteOwner": "This video exceeds your {{limit}} minute video limit. To upload this video, <1>upgrade now</1>.",
+	"UploadVideo_Error_Video_Duration_Visitor": "This viedo can’t be uploaded right now. Contact the site owner for more information.",
```
Comment (mharsat) [inline packages/common/web/statics/locale/messages_en.json:492]: Is this the format you agreed on with Adi? She said single video will be like total video error without the `limit`

Comment (mharsat) [reply packages/common/web/statics/locale/messages_en.json:492]: Also, isn't the format format for arguments `<1>arg</1>`? Or does `{{arg}}` also work

**Review by mharsat — COMMENTED** [review]
You still need the argument names from wix-ricos, I'll let you know when it's available

File: packages/common/web/statics/locale/messages_en.json (hunk 1 above)
Comment (mharsat) [reply packages/common/web/statics/locale/messages_en.json:492]: I'll pass the `limit` just in case

Comment (talevy17) [reply packages/common/web/statics/locale/messages_en.json:492]: I copied the text from the document as is and the limit was there.
the syntax for arguments in i18next is {{arg}}
<1>arg</1> is used to translate the first child of the Trans component, we use it to create the ugradeUrl links

Comment (mharsat) [reply packages/common/web/statics/locale/messages_en.json:492]: cool thanks

File: packages/common/web/statics/locale/messages_en.json [hunk 2]
```diff
@@ -465,6 +465,31 @@
 	"UploadFile_Error_Size_Item": "This file is too big",
 	"UploadFile_Error_StorageExceeded_Visitor": "Files can't be uploaded right now. Contact the site owner for more information.",
 	"UploadFile_Error_StorageExceeded_SiteOwner": "You've exceeded your site's storage limit. To save more media on your site, <1>upgrade now</1>.",
+	"UploadFile_Error_Mimetype_Mismatch": "This file's extension and file type don't match so it can’t be uploaded. Rename the extension and try again.",
```
Comment (oded-soffrin) [inline packages/common/web/statics/locale/messages_en.json:468]: this is not right place. see details here - https://wix.slack.com/archives/G016ZALEMU2/p1605178271008300

Comment (talevy17) [reply packages/common/web/statics/locale/messages_en.json:468]: Got it, I'll move it

File: packages/editor/web/src/RichContentEditor/Components/ErrorToast.jsx [hunk 3]
```diff
@@ -47,38 +37,23 @@ export default class ErrorToast extends Component {
...
     if (!isOpen) {
       return null;
     }
     const { isMobile } = this.context;
-    const errorMsg = this.getErrorMessage();
+
     return (
       <Suspense fallback={<div />}>
-        <Toast message={errorMsg} onClose={this.close} isMobile={isMobile} isError />
+        <ErrorMessageProvider
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/Components/ErrorToast.jsx:50]: In react we have context/provider. So using provider for something else is confusing.

Comment (shaulgo) [reply packages/editor/web/src/RichContentEditor/Components/ErrorToast.jsx:50]: This is just an ErrorMessage

**Review by shaulgo — APPROVED** [review]

**Pull request merged**